### ISSUE TITLE
SDL3, macOS: SDL_tray.h missing from framework bundle

### ISSUE THREAD

**Issue by andreasgrabher** [libsdl-org/SDL#11760]
I think the title says it all.

**Comment by madebr** [comment]
Indeed, dmg verification fails: https://github.com/madebr/SDL/actions/runs/12527498042/job/34941166857#step:6:71

**Comment by slouken** [comment]
I'll fix this.

**Comment by sezero** [comment]
CI failing:

```
CpHeader /Users/runner/work/SDL/SDL/Xcode/SDL/build/Release/SDL3.framework/Versions/A/Headers/SDL_tray.h /Users/runner/work/SDL/SDL/include/SDL_tray.h (in target 'SDL3' from project 'SDL')
    cd /Users/runner/work/SDL/SDL/Xcode/SDL
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -strip-unsigned-binaries -strip-deterministic -strip-tool /Applications/Xcode_15.2.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip -resolve-src-symlinks /Users/runner/work/SDL/SDL/include/SDL_tray.h /Users/runner/work/SDL/SDL/Xcode/SDL/build/Release/SDL3.framework/Versions/A/Headers
error: /Users/runner/work/SDL/SDL/include/SDL_tray.h: No such file or directory (in target 'SDL3' from project 'SDL')
```

**Comment by slouken** [comment]
Fixing...

**Comment by slouken** [comment]
Okay, this should be fixed in 01c360574
